conferences/migrations/3_add_sponsors.up.sql

```diff
@@ -0,0 +1,160 @@
+BEGIN;
+
+CREATE TYPE sponsorship_level AS ENUM ('platinum', 'gold', 'silver', 'bronze');
+
+CREATE TABLE sponsor(
+  id SERIAL PRIMARY KEY,
+  name TEXT NOT NULL,
+  address TEXT NOT NULL,
+  website TEXT NOT NULL,
+  sponsorship_level sponsorship_level NOT NULL
+);
+
+CREATE TYPE role AS ENUM ('marketing', 'logistics', 'technical', 'other', 'sole_contact');
+
+CREATE TABLE sponsor_contact_information(
+  id SERIAL PRIMARY KEY,
+  name TEXT NOT NULL,
+  role role NOT NULL, 
+  email TEXT NULL,
+  phone TEXT NULL,
+  sponsor_id SERIAL NOT NULL REFERENCES sponsor(id)
+);
+
+INSERT INTO sponsor (
+  name, 
+  address, 
+  website, 
+  sponsorship_level
+) VALUES (
+  'Google', 
+  'Google Plaza, Google Town, G00 G13', 
+  'https://www.google.com', 
+  'platinum'
+);
+
+INSERT INTO sponsor (
```

Comment:
Sponsors belong to conferences. Each year the list of sponsors may change, and the contact information for repeat sponsors may change.